server

```diff
@@ -295,11 +295,11 @@ main() {
     "amzn") amzn_install ;;
     "centos") centos_install ;;
     "rocky") centos_install ;;
-    "debian") debian_install ;;
+    "debian") deb_install ;;
```

Comment:
you can use `"debian" | "ubuntu")` and remove line 302

this can also be done for `centos` and `rocky` since they both use the same function

Reply:
Right! fixed.